test/node/conversation-archived.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let client2;
+let user2;
+let conversation;
+describe('Conversation Archived', () => {
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+        client2 = new Circuit.Client(config.bot2);
+        user2 = await client2.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+        await client2.logout();
+    });
+
+    it('should create a group conversation and raise a conversationCreated event', async () => {
+        const topic = `${Date.now()}a`;
+        const res = await Promise.all([
+            client.createGroupConversation([user2.userId], topic),
+            helper.expectEvents(client, [{
+                type: 'conversationCreated',
+                predicate: evt => evt.conversation.topic === topic && evt.conversation.participants.includes(user.userId) && evt.conversation.participants.includes(user2.userId)
+            }])
+        ]);
+        conversation = res[0];
+        assert(conversation && conversation.participants.includes(user.userId) && conversation.participants.includes(user2.userId));
+    }); 
+
+    it('should archive the conversation', async () => {
+        await client.archiveConversation(conversation.convId);
+        await helper.sleep(3000);
```

Comment:
Yes, if I remove it the test fails frequently, I think this is because getArchivedConversations uses getConversationsByFilter and maybe it runs into the same issue as when I was using it to search the labels.